lib/CodeGen/CodeGenFunction.h

```diff
@@ -2288,10 +2288,13 @@ class CodeGenFunction : public CodeGenTypeCache {
   void EmitDynamicNonNullCheck(const Address BaseAddr, const QualType BaseTy);
   void EmitDynamicOverflowCheck(const Address BaseAddr, const QualType BaseTy, const Address PtrAddr);
   void EmitDynamicBoundsCheck(const Address PtrAddr, const BoundsExpr *Bounds);
+  void EmitDynamicBoundsCheck(const BoundsExpr* DynamicBounds, const BoundsExpr *SrcBounds);
```

Comment:
Please rename these parameters to match the names that I suggested for the definition.